### PR TITLE
Refactor: Update README.md users and companies section

### PR DESCRIPTION
**Pull request by keyur9**
Refactor: Update README.md users and companies section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -309,31 +325,31 @@ var companies []*models.CompanyModel
...
+  "mrr": 24000,
+  "demographics": map[string]interface{}{
+      "alexa_ranking": 500000,
+      "employee_count": 47,
   },
 }
 
 // Prepare company model
 companyA := models.CompanyModel{
 	CompanyId:		  "67890",	// The only required field is your company id
```
Comment (dkm199) [inline README.md]: do you need literalFieldValue("67890"), here? whats the reason behind changing to use address for constants

Comment (keyur9) [reply README.md]: Nope, it is not required. CompanyId is the required field in the CompanyModel while CompanyDomain is an optional field (type: *string). So we can't assign string value to CompanyDomain field, so we use literalFieldValue helper function to make it an optional value.

File: README.md [hunk 2]
```diff
@@ -147,30 +151,30 @@ var moesifOptions = map[string]interface{} {
...
+  "sales_info": map[string]interface{}{
+      "stage": "Customer",
+      "lifetime_value": 24000,
+      "account_owner": "mary@contoso.com",
   },
 }
 
 // Only UserId is required
 user := models.UserModel{
   UserId:  "12345",
```
Comment (dkm199) [inline README.md]: i guess same goes here

File: README.md [hunk 3]
```diff
@@ -201,30 +209,30 @@ var users []*models.UserModel
...
+  "sales_info": map[string]interface{}{
+      "stage": "Customer",
+      "lifetime_value": 24000,
+      "account_owner": "mary@contoso.com",
   },
 }
 
 // Only UserId is required
 userA := models.UserModel{
   UserId:  "12345",
```
Comment (dkm199) [inline README.md]: and here

File: README.md [hunk 4]
```diff
@@ -249,38 +257,42 @@ import (
...
+  "mrr": 24000,
+  "demographics": map[string]interface{}{
+      "alexa_ranking": 500000,
+      "employee_count": 47,
   },
 }
 
 // Prepare company model
 company := models.CompanyModel{
 	CompanyId:		  "67890",	// The only required field is your company id
```
Comment (dkm199) [inline README.md]: same

File: README.md (hunk 1 above)
Comment (dkm199) [reply README.md]: we changed companyDomain from required to optional now? or earlier it was missed?

Comment (keyur9) [reply README.md]: It was optional from the beginning but the readme was missing it.

Comment (dkm199) [reply README.md]: ok

**Pull request merged**